### ISSUE TITLE
[a11y] fix issues that lerna run test --no-bail generates

### ISSUE THREAD

**Issue by btopro** [elmsln/issues#666]
We have automated testing for accessibility thanks to @collinkleest . Now we need to start going through the projects that fail this test and ensuring we actually fix / address what it's saying is a problem. Some elements in our monorepo also throw code level issues and some tooling level (like the element can't be tested in the default config, for example: testing cms-hax / anything with mobx requires defining window.process if it is not defined, which it wont be by default).

**Comment by btopro** [comment]
this tooling is now working. These need fixed on a case by case basis now.
